src/styles/KonnectorSync.styl

```diff
@@ -0,0 +1,18 @@
+@import cozy-ui
+
+:local // @stylint ignore
+    .account-forceConnection
+        button
+            border 0
+            background transparent
+            color blue
+            padding 0
+            text-decoration underline
+            text-transform uppercase
+
+        .submitting::after
```

Comment:
I think that a `aria-busy` attribute to `true` on the button should allow us to not need this ruleset.